draft-ietf-httpbis-resumable-upload.md

```diff
@@ -653,6 +654,56 @@ Content-Type: application/json
 }
 ~~~
 
+C) The following example shows a client attempting to resume an upload with an incorrect offset. The server rejects the request with a `409 (Conflict)` status code and includes the correct offset in the `Upload-Offset` header field. The client then retries the request with the correct offset.
```

Comment:
The previous examples were all happy paths, and my preference is to keep it that way. If we want to include examples for exceptional cases, we should discuss what are the most common failures and why this particular case needs highlighting.